vars/pipelineVars.groovy

```diff
@@ -1,7 +1,7 @@
 class pipelineVars implements Serializable {
     String defaultMarker = "core"
 
-    String jenkinsSvcAccount = "jenkins"
+    String jenkinsSvcAccount = "default"
```

Comment:
Modify readme as well, where we mentioned to create jenkins service account with edit access.